README.md

```diff
@@ -84,3 +84,19 @@ A Bolt-On is an optional part of the web interface configuration. These can be e
       bolt_on.enabled = true
       bolt_on.save!
     ```
+
+## Embedded Console 
+
+1. Ensure Node.js v8.12 is installed
+
+   * Install [directly](https://nodejs.org/en/download/)
+
+   * Install using [Node Version Manager](https://github.com/creationix/nvm#installation-and-update)
+
+2. Clone the `develop` branch of the `flight-terminal-service` repo found [here](https://github.com/alces-software/flight-terminal-service/tree/develop)
+
+3. Make necessary adjustments to values within `lib/config.json`
+
+4. Run the service using `INTEGRATION=no-auth-localhost yarn run start`
+
```

Comment:
With the release of 0.1.8 of flight-terminal-service, instructions 2, 3 and 4 are outdated.  They should be

 2. Clone the `master` branch ...

 3. Follow the instructions given in the "Quick start" section of Flight Terminal Service.

 4. Run the service using `yarn run start` (or `INTERFACE=0.0.0.0 yarn run start` to have it listen on all interfaces).